all/pom.xml

```diff
@@ -126,6 +126,13 @@
               </artifactSet>
               <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
               <createSourcesJar>true</createSourcesJar>
+              <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                  <manifestEntries>
+                    <Automatic-Module-Name>apache.directory.ldap.api.all</Automatic-Module-Name>
```

Comment:
@solomax @elecharny should't the name here (and in other poms) start with `org.` prefix?

Reply:
I believe this is up to project maintainers :)))